src/main/java/Duke.java

```diff
@@ -15,27 +17,44 @@ public static void main(String[] args) {
         System.out.println("Hello! I'm Duke");
         System.out.println("What can i do for you?\n");
 
-
-
         Scanner input = new Scanner(System.in);
         ArrayList<String> list = new ArrayList<>();
         int i = 1;
+
         while (true){
             String words = input.nextLine();
-            if (words.equals("bye")){
+            Task t = new Task(words);
+
+            if (t.description.equals("bye")){
                 System.out.println("Bye! Hope to see you again");
                 break;
             }
-            if (words.equals("list")) {
-                for (String s : list) {
 
-                    System.out.println(i + "." + s );
-                    i++;
+            else if (t.description.equals("list")) {
+
+                System.out.println("Here are the tasks in your list:");
+                for (String s : list) {
+                    System.out.println(s);
                 }
             }
+            else if(t.description.contains("done")){
```

Comment:
else if(t.description.contains("done")) (should have a space here){
For if-else statements, please follow this
if (condition) {
statements;
} else if (condition) {
statements;
} else {
statements;
}
There should be a space between "if and" (condition) and space between (condition) and "{"